tests/lib/screenshot-testing/support/test-environment.js

```diff
@@ -153,6 +153,9 @@ TestingEnvironment.prototype.setupFixture = function (fixtureClass, done) {
         self.reload();
         self.addPluginOnCmdLineToTestEnv();
 
+        self.fixtureClass = fixtureClass;
+        self.save();
```

Comment:
I think this could be done in Fixture.php around line 240, this would allow for using the DI this way in system/integration tests too.